tests/api/test_schemas_json_load.py

```diff
@@ -366,6 +375,36 @@ def test_resource_type():
         ResourceTypeSchemaV1().load(invalid_subtype)
 
 
+def test_custom_resource_type(config, vocabulary_clear):
```

Comment:
Is there a fixture for modifying the application configuration for the duration of a single test? Forgetting to reset it at the end of the test was very annoying... :sweat:

Reply:
In invenio-circulation we had created this context manager https://github.com/inveniosoftware/invenio-circulation/blob/master/tests/helpers.py#L26 to address this issue. You can see an example of usage [here](https://github.com/inveniosoftware/invenio-circulation/blob/c3fcc1a4674d5df24221e3c068e8e0c07db68016/tests/test_loan_base_constraints.py#L65). Just note that this is working for "swapping" one config key per time but can easily be extended to an object :)